Assets/Puppet/PuppetTransmission.cs

```diff
@@ -127,6 +128,10 @@ public void TrackEventChildTransmission()
         }
         OverrideProperties(transmissionTarget);
         var childTransmissionTarget = transmissionTarget.GetTransmissionTarget(ResolveChildToken());
+        if (CollectDeviceId.isOn)
```

Comment:
Toggle should be disabled after first action. See native apps as reference

Reply:
:+1: